recipe_scrapers/americastestkitchen.py

```diff
@@ -0,0 +1,59 @@
+# mypy: allow-untyped-defs
+
+import functools
+import json
+import re
+
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, normalize_string
+
+
+class AmericasTestKitchen(AbstractScraper):
+
+    @classmethod
+    def host(cls, domain="americastestkitchen.com"):
+        return domain
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def description(self):
+        return self.schema.description()
+
+    def total_time(self):
+        return get_minutes(self._get_additional_details.get("recipeTimeNote", None))
```

Comment:
I think it's OK to omit the default value of `None` from the `.get(...)` method calls here, because that's already the default when `get` doesn't find an item in a dictionary object:

```suggestion
        return get_minutes(self._get_additional_details.get("recipeTimeNote"))
```

(this also applies to some of the other callsites in the scraper)